### ISSUE TITLE
Spacer block: height in pixels field label should be properly associated

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#6918]
The new "Spacer" block has a "height in pixels" field in the sidebar. This label should be properly associated with the field.
- labels are not just for visual purposes
- they give the control a name
- when clicked, they focus the associated control
- there's no need for an aria-label attribute if the `<label>` element is descriptive enough
